provisioning/resources/control-plane/restart_services.go

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2016-2017 Snowplow Analytics Ltd.
+ * All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache
+ * License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.
+ *
+ * See the Apache License Version 2.0 for the specific language
+ * governing permissions and limitations there under.
+ */
+
+package main
+
+import (
+  "os/exec"
+  "errors"
+)
+
+var streamCollectorInit = "snowplow_stream_collector"
+var streamEnrichInit = "snowplow_stream_enrich"
+var esLoaderGoodInit = "snowplow_elasticsearch_loader_good"
+var esLoaderBadInit = "snowplow_elasticsearch_loader_bad"
```

Comment:
are those used anywhere?

Reply:
Nop, I forget to remove them after moved them to config. Will remove them.